src/main/java/org/datadog/jenkins/plugins/datadog/DatadogUtilities.java

```diff
@@ -108,16 +107,64 @@ public static boolean isJobTracked(final String jobName) {
     return !DatadogUtilities.isJobBlacklisted(jobName) && DatadogUtilities.isJobWhitelisted(jobName);
   }
 
+  /**
+   * Retrieve the list of tags from the Config file if regex Jobs was checked
+   *  @param jobName - A string containing the name of some job
+   *  @return - A Map of values containing the key and value of each Datadog tag to apply to the metric/event
+   */
+  public static Map<String,String> getRegexJobTags(final String jobName) {
```

Comment:
I don't think `final` is needed here either. `final` would be needed if there would be an interface defining the method signature as such. `final` only prevent for params to be reassigned within the method itself and only for its scope. Thus, it's only really useful when defined in interfaces.